__tests__/plugin.test.ts

```diff
@@ -0,0 +1,58 @@
+import { describe, expect, it } from '@jest/globals'
+import qPlugin from '../src/tailwind-plugin'
+import postcss from 'postcss'
+import tailwindcss from 'tailwindcss'
+import { ContentConfig } from 'tailwindcss/types/config'
+import path from 'path'
+
+function run(testContent: ContentConfig) {
+  return postcss(
+    tailwindcss({
+      //is this how tailwind executes?
+      content: testContent,
+      plugins: [qPlugin]
+    })
+  ).process('@tailwind utilities;', {
+    from: undefined
+  })
+}
+
+describe('Plugin can add index.js to content', () => {
+  it('Is added to the tailwind config with files key', async () => {
+    const testContent: ContentConfig = {
+      files: ['./src/content.js', './stories/**/*'],
+      relative: false
+    }
+    const filesLength = testContent.files.length
+    const twConfig = await run(testContent) //.then(t => {
+    const [msgs, msgsLength] = [twConfig.messages, twConfig.messages.length]
+    expect(msgsLength).toBe(filesLength + 1)
+    expect(msgs.some(msg => msg.file.includes('index.js'))).toBe(true)
+  })
+  it('Can add to the content if array', async () => {
+    const testContent: ContentConfig = ['./src/content.js', './stories/**/*']
+    const filesLength = testContent.length
+    const twConfig = await run(testContent)
+    const [msgs, msgsLength] = [twConfig.messages, twConfig.messages.length]
+    expect(msgsLength).toBe(filesLength + 1)
+    expect(msgs.some(msg => msg.file.includes('index.js'))).toBe(true)
+  })
+  it('Uses the right path to the package if relative', () => {})
```

Comment:
Should this test be fleshed out?

Reply:
Addressed this, thanks for catching it :D